home/javascript/react/containers/FigureExpressionAntibodyTable.js

```diff
@@ -64,7 +64,7 @@ const FigureExpressionAntibodyTable = ({figureId}) => {
         <DataTable
             columns={columns}
             dataUrl={`/action/api/figure/${figureId}/antibody-labeling`}
-            rowKey={row => row.antibody.zdbID}
+            rowKey={() => Math.random()}
```

Comment:
Why use a random rowKey?

Reply:
Because the unique row id is a combination of multiple columns.